Makefile

```diff
@@ -0,0 +1,61 @@
+IMAGE_NAME = canopy
+DOCKERFILE_DIR = .
+COMMON_BUILD_ARGS = --progress plain
+EXTRA_BUILD_ARGS =
```

Comment:
Can you please make it possible to pass these as arguments directly to `make build`, not only as env vars?